wagtailmenus/tests/test_backend.py

```diff
@@ -186,22 +191,9 @@ def test_mainmenu_edit(self):
         menu_model.edit_handler = ObjectList(menu_model.content_panels)
         response = self.client.get('/admin/wagtailmenus/mainmenu/edit/1/')
 
-    def test_not_condensedinlinepanel(self):
-        self.assertTrue(isinstance(FlatMenuItemsInlinePanel(), InlinePanel))
-        self.assertTrue(isinstance(MainMenuItemsInlinePanel(), InlinePanel))
-
-    @modify_settings(INSTALLED_APPS={'append': 'condensedinlinepanel'})
-    def test_condensedinlinepanel(self):
-        from condensedinlinepanel.edit_handlers import CondensedInlinePanel
-        self.assertTrue(isinstance(FlatMenuItemsInlinePanel(), CondensedInlinePanel))
-        self.assertTrue(isinstance(MainMenuItemsInlinePanel(), CondensedInlinePanel))
-
     def test_mainmenu_edit_multisite(self):
         Site.objects.create(
-            hostname='test2.com', port=80, root_page_id=2,
```

Comment:
Shouldn't these lines still be there?

Reply:
No. The condensedinlinepanel tests are moved further down, and the extra sites being created in the other test are unnecessary.